### PR TITLE
Port `v` and `V` to RzShell

### PR DESCRIPTION
**Pull request by Rot127**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

This brings some changes:

- Visual commands passed to `V`, must be given as normal argument (space between command and string).
- Changes "load layout" `v [layout-name]` to `vl [layout-name]`, to have a hard distinction between `v` and `vl`.
- Change "store layout" `v= [layout-name]` tp `vs [layout-name]`. Because it is more descriptive and matches `vl` a little better.
- Drops the vi command for two reasons:
  - vi (open file in editor) is not at all related to the visual mode. Hence shouldn't be in the same group.
  - (Opinionated) Users can just open the file in another terminal window. Why go the extra way over Rizin?

**Test plan**

All green

**Closing issues**

...

### CODE REVIEW

**Review by wargio — APPROVED** [review]

File: librz/core/cmd_descs/cmd_interactive.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          - text: ""
+            arg_str: "Press: ?"
+            comment: Opens the help.
+          - text: ""
+            arg_str: "Press: p"
+            comment: Switches between different printing modes (disassembly, hex view, etc.).
+          - text: ""
+            arg_str: "Press: <TAB>"
+            comment: Toggles different configurations of the current printing mode.
+      - name: Examples
```
Comment (XVilka) [inline librz/core/cmd_descs/cmd_interactive.yaml]: Add more information what the argument means - a sequence of the pressed keys in the visual mode. Also, it's better to have a few examples for those cases.

**Review by XVilka — CHANGES_REQUESTED** [review]
In my practice, commands like `Vp`, `Vpp`, and `VP` are common, maybe add some aliases for those particular cases?

File: librz/core/cmd_descs/cmd_interactive.yaml (hunk 1 above)
Comment (XVilka) [reply librz/core/cmd_descs/cmd_interactive.yaml]: Also, this is not full help. Either implement a full help or make `V?` corresponding to the one press of `?` in the `V` mode, and `V??` corresponding to two presses of `?` in the `V` mode.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/core/cmd_descs/cmd_interactive.yaml (hunk 1 above)
Comment (Rot127) [reply librz/core/cmd_descs/cmd_interactive.yaml]: Then I go with `VH`/`VHH` for showing the help of the `V` mode. `V?` should be really just for the command.
But I'd like to have more details available to the user before they enter the V mode.
I remember I got confused my first time. It was the "How do I exit vim moment".

Comment (XVilka) [reply librz/core/cmd_descs/cmd_interactive.yaml]: Ok to have `VH` for the full help, but still examples section and explanation what the argument exactly is needed in the `V?` and `V??` output.

**Review by XVilka — APPROVED** [review]

**Review by wargio — APPROVED** [review]
i'm ok with these.

**Comment by XVilka** [comment]
@Rot127 please rebase one more time to resolve the conflict

**Pull request merged**